lib/maths/CTimeSeriesModel.cc

```diff
@@ -2619,18 +2549,13 @@ bool CMultivariateTimeSeriesModel::probability(const CModelProbabilityParams& pa
     };
     using TJointProbability2Vec = core::CSmallVector<SJointProbability, 2>;
 
-    auto updateJointProbabilities = [&](maths_t::EProbabilityCalculation calculation,
-                                        double value_, const TDouble10Vec2Vec& pl,
+    auto updateJointProbabilities = [&](const TDouble10Vec2Vec& pl,
```

Comment:
ditto

Reply:
Done in [5d3c3a3](https://github.com/elastic/ml-cpp/pull/721/commits/5d3c3a3b79319afb4949e5ad10666d9adf1036d3).